src/components/Queue/components/PrintCardContainer.jsx

```diff
@@ -1,32 +1,70 @@
-import React from 'react';
+import React, { useState, Fragment } from 'react';
 import './PrintCardContainer.css';
 import StatusDropdown from './components/StatusDropdown';
 import PrintDateAdded from './components/PrintDateAdded';
 
-const PrintCardContainer = ({data}) => {
-    const colorCircleStyle = {
-        backgroundColor: `${data.color}`
-    };
-
-    return (
-        <div className='printCardContainer'>
-            <div className='printFileName col20'>
-                <p>{data.fileName}</p>
-            </div>
-            <div className='col20'>
-                <div className='colorCircle' style={colorCircleStyle}></div>
-            </div>
-            <div className='submitDate col20'>
-                <PrintDateAdded data={data}/>
-            </div>
-            <div>
-                <StatusDropdown data={data}/>
-            </div>
-            <div className='printAdditionalInfo col20'>
-                <img alt='arrLogo'/>
-            </div>
-        </div>
-  );
+import { CirclePicker } from 'react-color';
+
+const PrintCardContainer = ({ data }) => {
+	const [circleColor, setcircleColor] = useState(data.color);
+	const [hoverState, setHoverState] = useState(false);
+
+	const colorCircleStyle = {
+		backgroundColor: `${circleColor}`
+	};
+
+	const handleChangeComplete = color => {
+		console.log('New color will be', color);
```

Comment:
Please delete console.logs, when they pile up it makes the console hard to read

Reply:
Deleted the console.logs